lua/curl/parser.lua

```diff
@@ -68,9 +68,13 @@ local format_command_for_curl = function(lines)
 		if found_first_json_char(json_nesting_stack, cleaned_line) then
 			opening_json_char = cleaned_line:sub(1, 1)
 			closing_json_char = opening_json_char == "[" and "]" or "}"
-			is_line_json_close(json_nesting_stack, opening_json_char, closing_json_char, cleaned_line)
-
 			cleaned_line = "'" .. cleaned_line
+
+			is_line_json_close(json_nesting_stack, opening_json_char, closing_json_char, cleaned_line)
```

Comment:
This function returns a boolean whether the line contains a json closer, however the result is never used. 
Is the case fixed if you do the same as in the else branch instead?
```lua